testing_database/creating-the-testing-database.md

```diff
@@ -0,0 +1,34 @@
+# Creating the testing database
```

Comment:
I'm not sure if creating a new "testing_database" folder is the right way to go here. Perhaps we could include this as an extended doc string in the test_db_migration integration test? I'd be interested in others feedback on this before we decide which way to go.

Reply:
Discussed this with @gregory-schiano  on architecture hours and decided to create a new folder but we can discuss again.